CMakeLists.txt

```diff
@@ -59,6 +59,16 @@ endif(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 # RPP Default Options
 set(DEFAULT_BUILD_TYPE "Release")
 
+# RPP Audio support - (default = ON, support currently on Ubuntu - user to set to OFF otherwise)
+if(NOT DEFINED RPP_AUDIO_SUPPORT)
```

Comment:
is this an option for the user?
please give it as option in cmake then similar to this https://github.com/ROCm/MIVisionX/blob/fcb9876138d9bb7389785d4e958266d84627b824/CMakeLists.txt#L51

Reply:
yeah, modified this to use option() and added some more support